src/test/java/io/appium/java_client/ios/IOSGesturesTest.java

```diff
@@ -33,6 +34,14 @@
         driver.tap(2, e, 2000);
         assertEquals(driver.findElementByXPath("//*[@name = \"Answer\"]").getText(), "6");
     }
+    
+    @Test public void doubleTapTest() {
+        IOSElement firstField = (IOSElement) driver.findElementById("IntegerA");
+        firstField.sendKeys("2");
+        driver.doubleTap(firstField);
+        IOSElement editingMenu = (IOSElement) driver.findElementByClassName("UIAEditingMenu");
+        assertNotNull(editingMenu);
```

Comment:
there was little to double tap on in ios TestApp. But found the editing menu shows up when you double tap on one of the fields.